src/promptflow-devkit/promptflow/_sdk/entities/_run.py

```diff
@@ -661,6 +662,15 @@ def _to_rest_object(self):
             start_time = self._created_on.isoformat() + "Z" if self._created_on else None
             end_time = self._end_time.isoformat() + "Z" if self._end_time else None
 
+            # extract properties that needs to be passed to the request
+            total_tokens = self.properties[FlowRunProperties.SYSTEM_METRICS].get("total_tokens", 0)
+            properties = {
+                Local2CloudProperties.TOTAL_TOKENS: total_tokens,
+                Local2CloudProperties.EVAL_RUN: self.properties.get(Local2CloudProperties.EVAL_RUN, None),
```

Comment:
maybe we can have a constants of all the accepted properties list, and handle it more generally?

Reply:
Updated.